go.sum

```diff
@@ -1,4 +1,3 @@
-cel.dev/expr v0.15.0/go.mod h1:TRSuuV7DlVCE/uwv5QbAiW/v8l5O8C4eEPHeu7gf7Sg=
```

Comment:
not sure why all these deletions.

Reply:
Yeah sorry, The changes to `go.sum` were unintentional. They were caused by running Go tests locally, which sometimes updates the `go.sum` file to ensure dependency integrity. I've since reverted `go.sum` to its previous state and excluded it from this PR, as these changes are not related to the main purpose of this update,